### PR TITLE
Create a Migration Script/brach 3

### PR DESCRIPTION
**Pull request by Jagrmi-C**
ALEMBIC
Create a Migration Script
error!!!
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) FATAL:  password authentication failed for user "postgis"

### CODE REVIEW

File: alembic.ini [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# to alembic/versions.  When using multiple version
+# directories, initial revisions must be specified with --version-path
+# version_locations = %(here)s/bar %(here)s/bat alembic/versions
+
+# the output encoding used when revision files
+# are written from script.py.mako
+# output_encoding = utf-8
+
+;sqlalchemy.url = driver://user:pass@localhost/dbname
+sqlalchemy.url = postgresql://postgis:faust098@localhost:5432/belarus_ma
```
Comment (Jagrmi-C) [inline alembic.ini]: Here I import the settings data from the file setting.py?

**Comment by Jagrmi-C** [comment]
"did you run migrations yourself"
No
"""
E:\PYTHON\OSM\osm-validator>alembic upgrade head
usage: alembic [-h] [host_port]
alembic: error: unrecognized arguments: head
"""

File: alembic.ini [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# to alembic/versions.  When using multiple version
+# directories, initial revisions must be specified with --version-path
+# version_locations = %(here)s/bar %(here)s/bat alembic/versions
+
+# the output encoding used when revision files
+# are written from script.py.mako
+# output_encoding = utf-8
+
+;sqlalchemy.url = driver://user:pass@localhost/dbname
+sqlalchemy.url = postgresql://{}:{}@{}/{}format(settings.DATABASE['user'], settings.DATABASE['password'], settings.DATABASE['host'], settings.DATABASE['database'])
```
Comment (tbicr) [inline alembic.ini]: lets remove this two lines - you set in env.py

File: alembic/env.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+config.set_main_option('sqlalchemy.url', "postgresql://{}:{}@{}/{}".format(
+                           DATABASE['user'],
+                           DATABASE['password'],
+                           DATABASE['host'],
+                           DATABASE['database']))
+# config.set_main_option('sqlalchemy.url', "postgresql://jagrmi:1989@localhost/faust")
```
Comment (tbicr) [inline alembic/env.py]: lets remove this comment

File: alembic/versions/b95b832cb0c4_create_user_table.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+# revision identifiers, used by Alembic.
+revision = 'b95b832cb0c4'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'UserOSM',
```
Comment (tbicr) [inline alembic/versions/b95b832cb0c4_create_user_table.py]: lets rename to `user`

File: models.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    def __repr__(self):
+        return "<UserOSM({},{})>".format(self.osm_uid, self.osm_user)
+
+
+def run():
+    # engine = create_engine('postgresql://jagrmi:1989@localhost/faust')
+    engine = create_engine(postgresql)
+    metadata = Base.metadata
+    metadata.create_all(engine)
+    return Base
```
Comment (tbicr) [inline models.py]: this function doesnot used

File: models.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+async def setup(app):
+    engine = create_engine(postgresql)
+    metadata = Base.metadata
+    metadata.create_all(engine)
```
Comment (tbicr) [inline models.py]: this line will create tables, but you have migrations instead, so your method can look like:

    async def setup(app):
        engine = create_engine(postgresql)
        app['db_engine'] = engine
        app['db_declarative_base'] = Base

File: settings.py [hunk 7]
```diff
@@ -25,6 +25,6 @@
 }
 
 try:
-    from settings_local import *
+    from local_settings import *
```
Comment (tbicr) [inline settings.py]: lets leave `settings_local.py`

**Pull request merged**